app/main.py

```diff
@@ -1 +1,12 @@
-# write your code here
+def copy_file(command: str) -> None:
+    operation, file_name, destination = command.split()
+
+    try:
+        if operation != "cp" or file_name == destination:
+            print("File error")
```

Comment:
You don't need this print, better first check file_name != destination and operation == "cp" and call your context maneger without any `else`